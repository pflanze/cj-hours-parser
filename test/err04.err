Error: can't process file "test/err04.txt"

Caused by:
    premature EOF in a multi-line comment started on line 2, missing 1 ']'
