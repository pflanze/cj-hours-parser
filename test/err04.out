item=SingleLineComment(Owned(" for all we know"))
    =SingleLineComment(Owned(" for all we know"))
